src/provider/p_ibmca.h

```diff
@@ -124,7 +124,9 @@ struct ibmca_key {
     union {
         struct {
             size_t bits;
-            ica_rsa_key_crt_t private;
+            size_t keylength;
```

Comment:
A key can have either only the CRT sub-structure filled out or the only the ME sub-structure, or both. 
At many places I juts need the RSA key length in bytes. Previously I was just using the key_length form the CRT substructure because this was the only one and was always filled out. 
Now I would need to check if the CRT substructure is filled out, and only then I can take key_length from there, otherwise I need to use key_length from the ME substructure. To avoid all this hassle at many places, I added the outer keylength which is always filled out, regardless of the sub-structures.